src/test/java/service/ExaminerServiceImplTest.java

```diff
@@ -0,0 +1,45 @@
+package service;
+
+import org.example.model.Question;
+import org.example.service.ExaminerServiceImpl;
+import org.example.service.JavaQuestionService;
+import org.example.service.MathQuestionService;
+import org.example.service.QuestionService;
+import org.junit.Test;
```

Comment:
```suggestion
import org.junit.jupiter.api.Test;
```
Не ту аннотацию используешь.